src/transaction/TransactionController.ts

```diff
@@ -82,10 +82,23 @@ export interface Transaction {
   estimatedBaseFee?: string;
 }
 
+/**
+ * @type GasPriceValue
+ *
+ * GasPriceValue representation
+ * @property gasPrice - Value of the gas price
```

Comment:
Nit: What format is the gas price expected to be in? Is it a hex string? Is it 0x-prefixed? We may as well be specific if we're adding a description. Format confusion has been responsible for many bugs.

Reply:
Fixed! Its a hex string, I also added it to `FeeMarketEIP1559Values`.

Obs: It looks like we are not using `GasPriceValue`, I didn't see it in the mobile code, I'll confirm this with @rickycodes tomorrow

Reply:
Yes, I assumed the same thing, but for non-EIP-1559 transactions that parameter is omitted. 

```javascript
const stopTransaction = useCallback(() => {
    safelyExecute(() => Engine.context.TransactionController.stopTransaction(tx?.id));
}, [safelyExecute, tx]);
```
```javascript
const speedUpTransaction = useCallback(() => {
    safelyExecute(() => Engine.context.TransactionController.speedUpTransaction(tx?.id));
}, [safelyExecute, tx]);
```